integration/src/main/java/ai/djl/integration/tests/ndarray/NDArrayOtherOpTest.java

```diff
@@ -727,10 +728,22 @@ public void testArgMin() {
     @Test
     public void testEncodeDecode() {
         try (NDManager manager = NDManager.newBaseManager()) {
-            NDArray array = manager.create(new long[] {0, 3, 4, 2}, new Shape(2, 2));
+            NDArray array = manager.create(new byte[] {0, 3}, new Shape(2));
```

Comment:
is there any reason to change the dtype to int8 here?

Reply:
No, just trying to reduce the size of encoded buffer in the test.